src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,53 @@
-export const Tabs = () => {};
+export type Tab = {
+  id: string
+  title: string
+  content: string
+};
+
+type TabsProps = {
+  tabs: Tab[]
+  selectedTabId: string
+  onTabSelected: (tab: Tab) => void
+};
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }: TabsProps) => {
+  let selectedTabIdCopy = selectedTabId;
+
+  if (tabs.length > 0 && !tabs.map(tab => tab.id).includes(selectedTabId)) {
+    selectedTabIdCopy = tabs[0].id;
+  }
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            const isTabSelected = tab.id === selectedTabIdCopy;
```

Comment:
think this is redundant, please create separate function for this logic to be, like :

```
 const selectedTab = tabs
    .find(({ id }) => id === selectedTabId) || tabs[0];
```

Reply:
I created function.